JSDemos/Demos/Tabs/Selection/Angular/app/app.service.ts

```diff
@@ -6,54 +6,38 @@ export class Tab {
   text: string;
 
   icon: string;
-
-  content: string;
 }
 
-export class Longtab {
-  text: string;
-}
-
-const tabs: Tab[] = [
+const employees: any = [
   {
     id: 0,
-    text: 'user',
     icon: 'user',
-    content: 'User tab content',
+    text: 'John Heart',
+    position: 'CEO',
+    picture: 'https://js.devexpress.com/Demos/WidgetsGallery/JSDemos/images/employees/01.png',
+    notes: 'John has been in the Audio/Video industry since 1990. He has led DevAv as its CEO since 2003. When not working hard as the CEO, John loves to golf and bowl. He once bowled a perfect game of 300.',
   },
   {
     id: 1,
-    text: 'comment',
-    icon: 'comment',
-    content: 'Comment tab content',
+    icon: 'user',
+    text: 'Olivia Peyton',
+    position: 'Sales Assistant',
+    picture: 'https://js.devexpress.com/Demos/WidgetsGallery/JSDemos/images/employees/09.png',
+    notes: 'Olivia loves to sell. She has been selling DevAV products since 2012.  Olivia was homecoming queen in high school. She is expecting her first child in 6 months. Good Luck Olivia.',
   },
   {
     id: 2,
-    text: 'find',
-    icon: 'find',
-    content: 'Find tab content',
+    icon: 'user',
+    text: 'Robert Reagan',
+    position: 'CMO',
+    picture: 'https://js.devexpress.com/Demos/WidgetsGallery/JSDemos/images/employees/03.png',
```

Comment:
Replace the links, please like this: [https://github.com/DevExpress/devextreme-demos/blob/23_2/JSDemos/Demos/DataGrid/MultipleRecordSelectionAPI/Angular/app/app.service.ts#L31].

Reply:
sure, missed that